<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadsForm</class>
 <widget class="QFrame" name="DownloadsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>354</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="topFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../resources/resources.qrc">:/images/downloads29.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leSearch">
        <property name="placeholderText">
         <string>Search videosâ€¦</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <!--item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item-->
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>Clear list</string>
        </property>
       </widget>
      </item>
      <!--item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item-->
      <item>
       <widget class="QPushButton" name="btnPasteURLs">
        <property name="text">
         <string>Paste URLs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PreviewPanelButton" name="manageLabel">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="../../resources/resources.qrc">:/previewpanel/left_default</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="downloadsTreeView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionOpenFile">
   <property name="text">
    <string>Play file</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>Show in folder</string>
   </property>
  </action>
  <action name="actionSelectResolution">
   <property name="text">
    <string>Select resolution</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="text">
    <string>Remove from list</string>
   </property>
   <property name="toolTip">
    <string>Clear list</string>
   </property>
  </action>
  <action name="actionPauseSelected">
   <property name="text">
    <string>Pause selected</string>
   </property>
   <property name="toolTip">
    <string>Pause selected</string>
   </property>
  </action>
  <action name="actionResumeSelected">
   <property name="text">
    <string>Resume selected</string>
   </property>
   <property name="toolTip">
    <string>Resume selected</string>
   </property>
  </action>
  <action name="actionStopSelected">
   <property name="text">
    <string>Stop selected</string>
   </property>
   <property name="toolTip">
    <string>Stop selected</string>
   </property>
  </action>
  <action name="actionShowInLibrary">
   <property name="text">
    <string>Show in Library</string>
   </property>
   <property name="toolTip">
    <string>Show selected video in the Library tab</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewPanelButton</class>
   <extends>QLabel</extends>
   <header>previewpanelbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
